09-react.md

```diff
@@ -100,10 +100,183 @@ Al igual que en `xml`, si una etiqueta no tiene hijos, puede **autocerrarse**, p
+
+### **Llamar a un componente desde otro**
+
+Ahora que ya tienes tu componente creado, ya puedes utilizarlo desde otro componente llamandolo de la siguiente manera:
+
+```JS
+    /* /components/Welcome es la ruta en donde se encuentra tu componente creado*/
+
+    import Welcome from '/components/Welcome'
+
+    <Welcome />
+```
+
+Listo de esa forma ya puedes usar tu componente Welcome en donde quieras, y lo mejor de todo es que si haces un cambio en el componente se verá reflejado en todos los lugares en donde lo estes usando.
+
+Recuerda que un componente no tiene porque ser tan simple como el ejemplo, puedes agregar toda la lógica que necesites dentro de el, al igual que todo el contenido en JSX que desees.
+
 ## Ciclo de vida de componentes
 
 ## Props
 
+### **¿Que son los props?** 
+
+Los props no son más que variables que puedes enviar desde un componente padre a un componente hijo.
+
+Estos props pueden contener cualquier tipo:
+
+- Texto
+- Numero
+- Arreglos
+- Booleanos
+- Objetos 
+
+Veamos un ejemplo de como puede un componente padre puede enviar props a su componente hijo.
+
+#### **Componente padre**
+
+```JS
+    import Hijo from './Hijo'
+
+    const Padre = () => {
+        return (
+            <Hijo 
+                name = "Alexander"
+                age = {23}
+                favoriteColors = {["Blue", "Green", "Purple"]}
+                familyNames = {{
+                    father: "Mike",
+                    mother: "Rose",
+                    sister: "Emily"
+                }}
+            />
+        )
+    }
+
+    export default Padre
+```
+
+Veamos a profundidad que esta pasando en el componente padre:
+
+Antes que nada importamos el componente hijo `import Hijo from './Hijo'` al que le enviaremos los props desde el padre, recuerda que **'./Hijo'** es la ruta en donde esta creado el componente.
+
+Si quisieramos usar el componente hijo dentro del componente padre, simplemente lo usaríamos de esta forma: `<Hijo />` pero como nuestra intención en este ejemplo es pasarle props notaremos que tiene algunas cosas adicionales que no son nada más que nombres de variables (name, age, favoriteColors, familyNames) con sus respectivos valores.
+
+Hay algo muy importante en cada props que queremos enviar, y es que, todas estan encerradas entre **{ }** ( menos **name** que contiene un string ) y esto es para decirle a JSX que ese contenido lo debe de interpretar como JavaScript nativo y no como una cadena de texto. te estarás preguntando que pasa con las dobles {} en **familyNames** y la respuesta es simple:  
+Las llaves externas son las que usará JSX y las segundas es porque estas enviando un objeto de Javascript y un objeto se representa con {}.
+
+Ahora vamos a ver como recibir esas props en el componente hijo:
+
+#### **Componente hijo**
+
+```JS
+const Hijo = (props) => {
```

Comment:
NAB (Not A Blocker): Mantengamos todo código el inglés incluidos los nombres de funciones.